EuchreSolution/Euchre/GameScript/trick.ts

```diff
@@ -87,13 +88,14 @@ class Trick {
 		if (this.__playedCards.length === 0) {
 			this.__suitLead = card.suit;
 		}
-		this.__playedCards.push({ player: this.__currentPlayer, card });
-		this.removeFromHand(this.__currentPlayer, card);
+		this.__playedCards.push({ player: currentPlayer, card });
+		this.removeFromHand(currentPlayer, card);
 
-		animShowText(Player[this.__currentPlayer] + " played " + card.id, MessageLevel.Step, 1);
-		animPlayCard(this.__currentPlayer, card.id);
+		const message = `${Player[currentPlayer]} played ${getCardShorthand(card)}`;
+		animShowText(message, MessageLevel.Step, 1);
+		animPlayCard(currentPlayer, card.id);
 
-		this.__currentPlayer = getNextPlayer(this.__currentPlayer);
+		this.__currentPlayer = getNextPlayer(currentPlayer);
 		if (this.__alone && this.__currentPlayer === getPartner(this.__maker)) {
```

Comment:
Don't need this anymore since getNextPlayer handles it now

Reply:
It was taken out later, but I put it in the right spot now